test/run_translated_c.zig

```diff
@@ -96,4 +96,22 @@ pub fn addCases(cases: *tests.RunTranslatedCContext) void {
         \\  return 0;
         \\}
     , "");
+
+    cases.add("cast signed array index to unsigned",
+        \\#include <stdlib.h>
+        \\void main(int argc, char **argv) {
```

Comment:
I believe using `void main` rather than `int main` makes your exit code undefined. Better would be to use c_int and return 0 like the other tests do.

Reply:
good catch. done.